packages/docs/developer-resources/contractkit/migrating-to-contractkit-v1.md

```diff
@@ -0,0 +1,84 @@
+# Migrating to ContractKit v1.0.0
+
+cLabs recently released ContractKit version 1.0.0. This document explains the key differences and shows you how you can start using the updated package.
+
+If you are using a previous version of ContractKit, you can continue using that version and you will only need to make the following changes when you upgrade.
+
+The main benefit of using the new version include:
+ - Reduced bundle size
+ - Better typescript support
+ - Improved maintenance by making it easier to use other libraries
+
+## ContractKit packages
+
+ContractKit is now a [suite of packages](https://github.com/celo-org/celo-monorepo/tree/master/packages/sdk).
+
+### Main packages
+
+ - `Connect` depends on the `web3` package and allows you to specify the connection provider and contains some configuration information for signing transactions.
+ - `ContractKit` depends on `connect` and `wallet-local` and allows you to connect to the Celo network, locally sign transactions and has more configuration options than `connect`. It also contains a registry of [Celo core contracts](contracts-wrappers-registry.md) to make it easier to interact with them.
```

Comment:
> This is a reduced subset of the previous contractKit. This is the layer in charge of loading and using our core contracts. Internally, uses the `connect` package described above.
> It has our contracts generated from the abis, their wrappers, and also the logic to make claims

Same as my above comment; some additional language that could be potentially useful to include!